src/app/_client-services/identity_service.ts

```diff
@@ -424,4 +424,198 @@ export class IdentityService {
 
 		return paymentInfo?.partialFee;
 	}
+
+	async getJudgementStats(): Promise<IJudgementStats> {
```

Comment:
if we are not using people chain api in this, please make it a util function and not in this api service class